vital/tests/test_mesh_io.cxx

```diff
@@ -0,0 +1,184 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief test core mesh_io functionality
+
+#include <kwiversys/SystemTools.hxx>
+#include <tests/test_gtest.h>
+#include <tests/test_scene.h>
+#include <arrows/core/mesh_operations.cxx>
```

Comment:
Probably best not to include .cxx files if we can avoid it, especially files from arrows in vital.  Can we just save a version of the triangulated cube mesh to a file and use that data?